### DISCUSSION TITLE
Interpretation and recalculation of predicted value

### DISCUSSION THREAD

**Discussion by SarosdiZs** [kogalur/randomForestSRC#281] [Q&A] [answered]
Hi,
I am just getting familiar with random survival trees based on [Ishwaran (2008)](https://arxiv.org/pdf/0811.1645.pdf) and I have a problem with the predicted value. I set up a dummy database and fitted a survival forest of a single tree. I was wondering what are the  highlighted values in the terminal nodes, below the number of in-bag instances. I found in the documentation that it must be some expected number of deaths (events of interest). I tried to recalculate them based on the cited document, but I got totally different result. I attached my very simple code. I would really appreciate any kind of help!
Thank you in advance!

[random_experimenting.txt](https://github.com/kogalur/randomForestSRC/files/9021781/random_experimenting.txt)
Where am I wrong and how is the value 1.78 on the pic is obtained?

<img width="413" alt="question" src="https://user-images.githubusercontent.com/97222513/176731872-999f2237-0d8d-45ec-a0eb-0b6c2c2ce76e.png">

**Comment by ishwaran** [comment, marked as answer]
By default, the tree plots the `oob` mortality values, which you can see by the following:

```
> as.numeric(na.omit(unique(rsf_model$predicted.oob)))
[1]  1.775383 37.333333 21.000000 11.000000

```
The mortality values are obtained from cumulative sum of the CHF (and in this case from the `oob` CHF) which you can see from 

```
> as.numeric(na.omit(unique(rowSums(rsf_model$chf.oob))))
[1]  1.775383 37.333333 21.000000 11.000000

```

Here is also a quick way to verify the inbag sample sizes:

```
> m=rsf_model$membership[rsf_model$inbag==1]
> tapply(m,m,length)
 1  2  3  4 
 5  7  8 43 
```
